docs/source/models_and_datasets.md

```diff
@@ -17,4 +17,15 @@ limitations under the License.
 
 # Morpheus Models
 
-Morpheus comes with a number of pretrained models with corresponding training, validation scripts, and datasets. The latest release of these models can be found [here](https://github.com/nv-morpheus/Morpheus/blob/-/models).
\ No newline at end of file
+Morpheus comes with a number of pretrained models with corresponding training, validation scripts, and datasets. The latest release of these models can be found [here](https://github.com/nv-morpheus/Morpheus/blob/-/models).
+
+|Model|GPU Mem Req|Description|
+|-----|-----------|-----------|
+|[Anomalous Behavior Profiling (ABP)](https://github.com/nv-morpheus/Morpheus/blob/-/models#anomalous-behavior-profiling-abp)|2015MiB|This model is an example of a binary classifier to differentiate between anomalous GPU behavior such as crypto mining / GPU malware, and non-anomalous GPU-based workflows (e.g., ML/DL training). The model is an XGBoost model.|
```

Comment:
We've been discouraged from using Latin abbreviations like "e.g." and "i.e." in favor of spelling out "For example"